cli/cmd/internal/client/install.go

```diff
@@ -33,15 +32,8 @@ var CmdInstall = &cobra.Command{
 }
 
 func init() {
-	pf := CmdInstall.PersistentFlags()
-
-	argToEnv := map[string]string{}
-
-	cmd.KubeConfigFlags(pf, argToEnv)
-	cmd.AddEnvToUsage(CmdInstall, argToEnv)
-
-	CmdInstall.Flags().BoolP("verbose", "v", true, "Wether to print logs to stdout")
-	CmdInstall.Flags().BoolP("interactive", "i", false, "Whether to ask the user or not")
+	CmdInstall.Flags().BoolP("verbose", "v", true, "Whether to print logs (stdout) or not")
```

Comment:
Since you are at it, the description should say something like:

```
Print more detailed messages
```

because we do print logs to stdout anyway.

Reply:
Ok. Will do. Just a moment.

Reply:
Done with commit 72726e959cd047b11a6b487091306ad48b80d7a5